src/main/java/com/redhat/exhort/integration/trustedcontent/TrustedContentIntegration.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright 2023 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.redhat.exhort.integration.trustedcontent;
+
+import java.util.*;
+
+import org.apache.camel.Exchange;
+import org.apache.camel.LoggingLevel;
+import org.apache.camel.builder.endpoint.EndpointRouteBuilder;
+import org.apache.camel.model.dataformat.JsonLibrary;
+
+import com.redhat.exhort.api.PackageRef;
+import com.redhat.exhort.integration.Constants;
+import com.redhat.exhort.model.DependencyTree;
+
+import io.quarkus.runtime.annotations.RegisterForReflection;
+
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.ws.rs.core.MediaType;
+
+@ApplicationScoped
+@RegisterForReflection
+public class TrustedContentIntegration extends EndpointRouteBuilder {
+
+  @Override
+  public void configure() {
+
+    from(direct("recommendationsTrustedContent"))
+        .to(direct("extractPurls"))
+        .process(this::handleHeaders)
+        .to(direct("tcRequest"))
+        .transform()
+        .method(TcResponseHandler.class, "responseToMap");
+
+    from(direct("extractPurls")).process(this::extractPurlsFromDependencyTree);
+
+    from(direct("tcRequest"))
+        .marshal()
+        .json()
+        .setHeader(Exchange.HTTP_PATH, constant(Constants.TRUSTED_CONTENT_PATH))
+        .setHeader(Exchange.HTTP_METHOD, constant("POST"))
+        .setHeader(Exchange.CONTENT_TYPE, constant(MediaType.APPLICATION_JSON))
+        .circuitBreaker()
```

Comment:
it's missing the `faultToleranceConfiguration` with the timeout

Reply:
done.